samples/pmic/npm6001/boards/nrf52dk_nrf52832.overlay

```diff
@@ -0,0 +1,44 @@
+i2c0: &i2c0 {
+	compatible = "nordic,nrf-twim";
+	status = "okay";
+
+	pinctrl-0 = <&i2c0_default_alt_alt>;
+	pinctrl-1 = <&i2c0_sleep_alt_alt>;
+	pinctrl-names = "default", "sleep";
+	clock-frequency = <100000>;
+};
+
+&pinctrl {
+	i2c0_default_alt_alt: i2c0_default_alt_alt {
+		group1 {
+			psels = <NRF_PSEL(TWIM_SDA, 0, 26)>,
+				<NRF_PSEL(TWIM_SCL, 0, 27)>;
+			nordic,drive-mode = <NRF_DRIVE_S0D1>;
+			bias-pull-up;
+		};
+	};
+
+	i2c0_sleep_alt_alt: i2c0_sleep_alt_alt {
+		group1 {
+			psels = <NRF_PSEL(TWIM_SDA, 0, 26)>,
+				<NRF_PSEL(TWIM_SCL, 0, 27)>;
+			nordic,drive-mode = <NRF_DRIVE_S0D1>;
+			bias-pull-up;
```

Comment:
why is this needed in LP mode?

Reply:
The drive-mode property is removed now. Still want to retain pull ups in all states